### PR TITLE
Mention TROUBLESHOOTING.md in issue template

### PR DESCRIPTION
**Pull request by jtattermusch**
Better logs should make our lives easier.

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ If possible, provide a recipe for reproducing the error. Try being specific and
 ### What did you see instead?
  
 Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).
+
+See https://github.com/grpc/grpc/blob/master/TROUBLESHOOTING.md for how to diagnose problems better.
```
Comment (mehrdada) [inline .github/ISSUE_TEMPLATE.md]: I'd say change the link to

```
See [TROUBLESHOOTING.md](https://github.com/grpc/grpc/blob/master/TROUBLESHOOTING.md) for how to diagnose problems better.
```

Comment (mehrdada) [reply .github/ISSUE_TEMPLATE.md]: Nevermind actually. issue template won't be rendered properly

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by jtattermusch** [comment]
Known failures:
https://github.com/grpc/grpc/issues/16790
https://github.com/grpc/grpc/issues/15599
https://github.com/grpc/grpc/issues/16579

**Pull request merged**